app/assets/stylesheets/provider/_service_access.scss

```diff
@@ -15,6 +15,7 @@ form.formtastic fieldset > ol > li fieldset {
       .FeatureAccessList-item.FeatureAccessList-item--monitoring::after,
       .FeatureAccessList-item.FeatureAccessList-item--partners::after,
       .FeatureAccessList-item.FeatureAccessList-item--plans::after,
+      .FeatureAccessList-item.FeatureAccessList-item--policy_registry::after,
```

Comment:
Could you briefly explain how this works? how the changes in this CSS make another line appear in the screen?

Reply:
:open_mouth:

Reply:
I don't truly understand what are these lines for. But looks better.

Reply:
@akostadinov Well, it just indicates, that when there is a line - it means that the permission is related to services, and the permission only applies to the services that are currently selected.
For the permissions that have no line - it means that it doesn't matter which services are selected or not.

Reply:
Im' not cnovinced these lines make this really clear. Maybe it would be clearer just to have text "(for the selected services)". But it's fine for me also like this.